Client/mods/deathmatch/logic/CClientVehicle.cpp

```diff
@@ -2638,6 +2636,8 @@ void CClientVehicle::Create()
         if (m_pDriver)
             m_pDriver->WarpIntoVehicle(this, 0);
 
+         m_pVehicle->SetEngineOn(m_bEngineOn);
```

Comment:
why do we move this from top to bottom?

Reply:
> why do we move this from top to bottom?

to override engine setting on m_pDriver->WarpIntoVehicle(this, 0); it will set engine ON